pages/instructorAssessmentGroups/instructorAssessmentGroups.js

```diff
@@ -198,8 +199,9 @@ router.post('/', function (req, res, next) {
         res.locals.errormsg +=
           'Failed to remove ' + failedUids + '. Please check if the uid exist.\n';
       }
+    })().then(() => {
       obtainInfo(req, res, next);
-    })();
+    });
```

Comment:
This page is already a mess, and ideally should be rewritten from scratch (check #4857 among other issues). I updated it only enough to make it work.